viz-lib/src/visualizations/chart/plotly/updateAxesInversion.js

```diff
@@ -23,7 +22,7 @@ export default function updateAxesInversion(seriesList, layout, options) {
         }
         if (isObject(yaxis2)) {
           // TODO add xaxis2
-          updates.yaxis2 = undefined;
+          updates.yaxis2 = null;
```

Comment:
Likely due to nullish coalescing, undefined won't work here.
This erases the left Y axis.